desk/lib/bolt.hoon

```diff
@@ -63,35 +60,35 @@
     ?>  =(src.bowl our.bowl)
     =/  =bean  !<(bean vase) 
     ?-  -.bean
-        %toggle-which
-      =.  which  switch.bean
-      :-  ~[(emit which/b+which)]
-          this 
```

Comment:
This kind of indentation is only used when you are working within runes that are closed using `==` or `--`. Otherwise you always backstep.